Studio metrics revamp (#546)
apollographql/docs: src/content/graphos/basics/quickstart/self-hosted.mdx

> While we're here, execute some test queries against the supergraph! We'll look at metrics for those test queries in the next step.

Reviewer: ```suggestion
Let's execute some test queries against the supergraph. We'll look at metrics for those test queries in the next step.
```

## 6. View operation and field metrics

Because our router is connected to GraphOS, it automatically collects and reports metrics on incoming operations and their fields. You can then visualize those metrics in Studio.

Return to [GraphOS Studio](https://studio.apollographql.com?referrer=docs-content) and go to your supergraph variant's **Insights** page, which looks like this:

<img
  src="../../img/insights-quickstart.jpg"
  className="screenshot"

>   alt="The Insights page in GraphOS Studio"

Reviewer: ```suggestion
  alt="The Insights page in GraphOS Studio, showing metrics for operations and their fields"
```

/>

> The test queries you executed in the previous step should already be represented on this page. (If they aren't there yet, check the page again in a few minutes.) After you update all of your clients to query your router, the Insights page becomes vital to monitoring your supergraph's performance.

Reviewer: ```suggestion
Within a few minutes, the test queries you executed in the previous step should be represented on this page. The Insights page is vital to monitoring your supergraph's performance across all of your clients that query the router.
```
